Source/Blazorise/wwwroot/timePicker.js

```diff
@@ -57,6 +57,16 @@ export function initialize(element, elementId, options) {
         picker.altInput.disabled = options.disabled || false;
         picker.altInput.readOnly = options.readOnly || false;
         picker.altInput.placeholder = options.placeholder;
+
+        picker.altInput.addEventListener("blur", (e) => {
+            const isInput = e.target === picker._input;
+
+            // Workaround for: onchange does not fire when user writes the time and then click outside of the input area.
+            if (isInput && picker.isOpen === false) {
+                picker.input.dispatchEvent(utilities.createEvent("change"));
```

Comment:
There's also this other option that's in the code base, that's not too verbose: `new Event("change")` still would be pretty good. 
![image](https://github.com/Megabit/Blazorise/assets/22283161/ce075fcb-b417-4883-9f83-e03019a86d03)
![image](https://github.com/Megabit/Blazorise/assets/22283161/fb864f36-7049-42c6-97b5-584a48432231)

Reply:
No I actually meant that the utility might not be necessary since we could just do `new Event("change")` Anyway it's fine.

Reply:
But eitherway, we should just have every single one of them use the same implementation.